test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1669,5 +1670,4 @@ def tearDownClass(cls):
                 ret = mom.load_configuration(cls.saved_file)
                 if not ret:
                     raise Exception("Failed to load mom's custom setup")
-        if cls.use_cur_setup:
```

Comment:
why is this condition removed? 
This file is saved in setupclass only when --use-current-setup option is used

Reply:
Hi @RKORavi , Code is already under the same if condition, this was redundant.

Reply:
Sorry my bad did not notice that, thanks for pointed that out.